rke/k8s_rke_versions.go

```diff
@@ -0,0 +1,89 @@
+package rke
+
+import "github.com/rancher/types/apis/management.cattle.io/v3"
+
+func loadRKEDefaultK8sVersions() map[string]string {
```

Comment:
Can we think of merging this method with loadK8sRancherVersions? So we can avoid defining the same k8s version in 2 multiple places